app/src/main/java/com/khinthirisoe/bakingapp/ui/ingredients/IngredientsActivity.kt

```diff
@@ -18,24 +18,18 @@ class IngredientsActivity : AppCompatActivity(), StepsAdapter.StepRecyclerViewCl
 
     private var ingredientsAdapter: IngredientsAdapter? = null
     private var stepsAdapter: StepsAdapter? = null
+    private var bakingRecipe: Recipe? = null
 
     private lateinit var bakingName: String
```

Comment:
@KhinThiriSoe you need to remove this `bakingName` if you're adding the whole Recipe object